plugins/development/test/development-ts.test.ts

```
@@ -0,0 +1,74 @@
+import fs from 'node:fs/promises';
+import { strict as assert } from 'node:assert';
+import { scheduler } from 'node:timers/promises';
+
+import { mm, type MockApplication } from '@eggjs/mock';
+import { beforeEach, afterEach, it, describe } from 'vitest';
+
+import { escape, getFilepath, DELAY } from './utils.ts';
```

Biome (2.1.2)

[error] Do not shadow the global "escape" property.

Consider renaming this variable. It's easy to confuse the origin of variables when they're named after a known global.

(lint/suspicious/noShadowRestrictedNames)